src/Microsoft.ML.Transforms/GcnTransform.cs

```diff
@@ -805,8 +795,53 @@ public override SchemaShape GetOutputSchema(SchemaShape inputSchema)
     }
 
     /// <summary>
-    /// Lp Normalizing estimator takes columns and normalizes them individually by rescaling them to unit norm.
+    /// Normalizes (scales) vectors in the input column to the unit norm. The type of norm that is used can be specified by the user.
     /// </summary>
+    /// <remarks>
+    /// <format type="text/markdown"><![CDATA[
+    ///
+    /// ###  Estimator Characteristics
+    /// |  |  |
+    /// | -- | -- |
+    /// | Does this estimator need to look at the data to train its parameters? | No |
+    /// | Input column data type | Vector of <xref:System.Single> |
+    /// | Output column data type | Vector of <xref:System.Single> |
+    ///
+    ///
+    /// The resulting <xref:Microsoft.ML.Transforms.LpNormNormalizingTransformer> normalizes vectors in the input column individually
+    /// by rescaling them to the unit norm.
+    ///
+    /// Let $x$ be the input vector, $n$ the size of the vector, $L(x)$ the norm function selected by the user.
+    /// Let $\mu(x) = \sum_i x_i / n$ be the mean of the values of vector $x$. The <xref:Microsoft.ML.Transforms.LpNormNormalizingTransformer>
+    /// performs the following operation on each input vector $x$:
+    ///
+    ///     $y = \frac{x - \mu(x)}{L(x)}$
```

Comment:
I am not sure if I should use the double dollar sign here?

Reply:
This looks fine to me, but the following may be better:

\begin{equation*}
<equation without $ signs>
\end{equation*}

---
In reply to: [277183775](https://github.com/dotnet/machinelearning/pull/3454#discussion_r277183775) [](ancestors = 277183775)

Reply:
Same for the equations for the other norm funstions

---
In reply to: [277185443](https://github.com/dotnet/machinelearning/pull/3454#discussion_r277185443) [](ancestors = 277185443,277183775)

Reply:
I think the \begin{equation*} would need the amsmath package available wherever the LaTeX is being rendered so maybe use \[ and \] around the equation instead (only if you decide to replace $, which I don't think is necessary).

Reply:
This is in a markdown block, so I think you need the $  $

Reply:
But single($equation$) or double ($$equation$$)?

I would like those to be centered. But it is a detail.